sc2reader/readers.py

```diff
@@ -1723,6 +1724,49 @@ def control_group_update_event(self, data):
             }[data.read_bits(2)](),
         )
 
+class GameEventsReader_38215(GameEventsReader_36442):
+
+    def command_event(self, data):
+        # this function is exactly the same as command_event() from GameEventsReader_36442
+        # with the only change being that flags now has 25 bits instead of 23.
```

Comment:
Yeah, it is kind of shitty how this bit of code turned out. I can never get up the motivation to rewrite for less repetition though.